### DISCUSSION TITLE
optymalizacja-lacza-internetowego-router-openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#115] [General]
# optymalizacja-lacza-internetowego-router-openwrt/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/optymalizacja-lacza-internetowego-router-openwrt/

**Comment by spexpl** [comment]
A ja właśnie walczę i nie mogę zwyciężyć w bufferbloatowymi pingami przy moim łączu. Co prawda walczę z HP620 i opesense, ale mam wrażenie iż problem nie kwi u mnie tylko u mojego ISP.
